### PR TITLE
bump go control plane

### PR DESCRIPTION
**Pull request by EItanya**
## Warning
---
Notice: Development on `master` has slowed in favor of `feature-rc1`, which is under active development. 
Here's how to handle this.

Checklist
- [ ] I have opened my PR against the `feature-rc1` branch
- [ ] If you'd like to backport this to `master` (i.e. small bug fixes, typically not new features), cherry-pick the change from the `feature-rc1` branch
- [ ] Remove the contents of this template from the PR
---

### Solution
Explain your changes here

### Context
Why were these changes made? Alternatives considered? Any additional context to add?

### CODE REVIEW

File: changelog/v1.1.0/go-control-plane.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+changelog:
```
Comment (yuval-k) [inline changelog/v1.1.0/go-control-plane.yaml]: also dep-bump for solo-kit

File: pkg/utils/usage/usage_readers.go [hunk 2]
```diff
@@ -44,8 +44,8 @@ func (d *DefaultUsageReader) GetPayload() (map[string]string, error) {
 	}
 
 	envoys := 0
-	requestsCount := uint64(0)
-	connectionsCount := uint64(0)
+	requestsCount := float64(0)
```
Comment (yuval-k) [inline pkg/utils/usage/usage_readers.go]: why this change?

File: projects/gloo/api/external/envoy/api/v2/core/health_check.proto [hunk 3]
```diff
@@ -169,19 +169,6 @@ message HealthCheck {
         string authority = 2;
     }
 
-    // Custom health check.
-    message CustomHealthCheck {
```
Comment (yuval-k) [inline projects/gloo/api/external/envoy/api/v2/core/health_check.proto]: why was this deleted?

File: projects/metrics/pkg/metricsservice/storage_client.go [hunk 4]
```diff
@@ -21,8 +21,8 @@ type StorageClient interface {
 }
 
 type EnvoyMetrics struct {
-	HttpRequests   uint64
-	TcpConnections uint64
+	HttpRequests   float64
```
Comment (yuval-k) [inline projects/metrics/pkg/metricsservice/storage_client.go]: why float?

File: pkg/utils/usage/usage_readers.go (hunk 2 above)
Comment (EItanya) [reply pkg/utils/usage/usage_readers.go]: The prometheus API backing that object changed to a float64, so this represents that value now

File: projects/metrics/pkg/metricsservice/storage_client.go (hunk 4 above)
Comment (EItanya) [reply projects/metrics/pkg/metricsservice/storage_client.go]: Same as above. The proto value is float64, and there's no good simple way to convert

File: projects/gloo/api/external/envoy/api/v2/core/health_check.proto (hunk 3 above)
Comment (EItanya) [reply projects/gloo/api/external/envoy/api/v2/core/health_check.proto]: I can readd it, I was getting so annoyed by it because it's very nested
